### ISSUE TITLE
Disconnected adapters are still shown

### ISSUE THREAD

**Issue by stdedos** [processhacker/processhacker#599]
My expectation is that they won't show while disconnected

![image](https://user-images.githubusercontent.com/133706/79688060-45e94280-8254-11ea-9d81-d2b77447ce2d.png)

**Comment by dmex** [comment]
Adding and removing graphs from the sysinfo window is not yet supported when they're visible. You have to close and reopen the window. 

There's also the issue of some types of adapters adding/removing the adapters multiple times a second moving the window and graphs up and down causing seizures.

**Comment by stdedos** [comment]
> Adding and removing graphs from the sysinfo window is not yet supported when they're visible. You have to close and reopen the window.

I don't understand this in full.

I'll just say here that the last time those adapters were used was maybe even 2 years ago.

> There's also the issue of some types of devices (e.g. USB) adding/removing the adapters multiple times a second moving the window and graphs up and down causing seizures.

I get if updating the graphs works only between PH invocations. But, as I said above, those adaptors have not been plugged in nice never.

It was just disorienting to have to manually add them each time.

**Comment by dmex** [comment]
> I don't understand this in full.

Disconnected devices are always hidden on the sysinfo window for both the network and disk devices. I assumed they were being removed when you closed the window - which is how this feature used to work previously.

> those adaptors have not been plugged in

Windows used to return an error when the adapter was disconnected which is what we used to hide the network adapter from sysinfo window but seems to have changed. I've added a second check for the adapter state which fixes the issue in the latest nightly 👍
